assets/onboarding/features/index.jsx

```diff
@@ -16,13 +18,27 @@ const Features = () => {
 	const [ selectedSlugs, setSelectedSlugs ] = useState( [] );
 	const { goTo } = useQueryStringRouter();
 
+	// Features data.
 	const {
 		stepData,
 		submitStep,
 		isSubmitting,
 		errorNotice,
 	} = useSetupWizardStep( 'features' );
 	const features = stepData.options;
+	const submittedSlugs = stepData.selected;
+
+	// Features installation.
+	const { submitStep: submitInstallation } = useSetupWizardStep(
+		'features-installation'
+	);
```

Comment:
@yscik, I invested some time trying to change it, but I decided to create an issue for that to solve in the future. So we can release and fix it later, okay?

The problem to do it is explained here: https://github.com/Automattic/sensei/issues/3214

Reply:
Oops! Actually the selects aren't unused. We're using the `isSubmitting` and `errorNotice` for this case too. We just didn't get it again because it's "global".

Reply:
Yeah this should be fine, not worth changing with the difficulties in testing.